modules/gdscript/language_server/gdscript_extend_parser.cpp

```diff
@@ -723,7 +723,7 @@ Dictionary ExtendGDScriptParser::dump_class_api(const GDScriptParser::ClassNode
 			} break;
 			case ClassNode::Member::ENUM: {
 				Dictionary enum_dict;
-				for (int j = 0; j < m.m_enum->values.size(); i++) {
+				for (int j = 0; j < m.m_enum->values.size(); j++) {
 					enum_dict[m.m_enum->values[i].identifier->name] = m.m_enum->values[i].value;
```

Comment:
Shouldn't be here `i` also changed to `j`?

Reply:
Yes indeed, good catch.

Reply:
Fixed by cf62289d24ff4a71513e55940011864e929e7bf8.